components/api_version/src/api_v1.rs

```diff
@@ -35,4 +35,35 @@ impl KvFormat for ApiV1 {
     fn encode_raw_value_owned(value: RawValue<Vec<u8>>) -> Vec<u8> {
         value.user_value
     }
+
+    fn convert_encoded_key_version_from(
```

Comment:
Is there any test covers it?

Reply:
yes, new unit tests are added in `components/api_version/src/lib.rs` to cover newly added methods.